data_registry/views.py

```diff
@@ -200,6 +200,29 @@ def download_export(request, id):
     )
 
 
+def datasets_metadata(request):
+    return JsonResponse(
+        list(
+            Collection.objects.filter(public=True, job__active=True)
+            .values(
+                "id",
+                "title",
+                "country",
+                "last_retrieved",
+                "retrieval_frequency",
+                "source_id",
+                "frozen",
+                "language",
+                "source_url",
+                "update_frequency",
+                "region",
+            )
+            .annotate(data_from=F("job__date_from"), data_to=F("job__date_to"))
```

Comment:
I think this assumes there is one active job (which there should be, but it's not guaranteed by the DB).

In `search` we need to do it a bit differently.

Reply:
Ah ok, I didn't notice the `visible` method exists. Thanks!